test/torch_optimizer_test.py

```diff
@@ -680,12 +697,13 @@ def test_optimizer_local_aggregation_duplicated(device, communication_type, kwar
                                                   num_steps_per_communication=J)
     elif communication_type == "gradient.allreduce":
         optimizer = bf.DistributedGradientAllreduceOptimizer(optimizer, model=model,
-                                                             backward_passes_per_step=J)
+                                                             num_steps_per_communication=J)
     else:
         raise ValueError("Communication_type under test is not expected.")
 
     # Train and test
     for _ in range(num_epochs):
-        local_aggregation_train(model, optimizer, train_dataloader, isCUDA, mini_batch_size)
```

Comment:
test_optimizer_local_aggregation_duplicated also has ATC optimizer test, turn on these tests. Check if other ATC local aggregation tests in this file exist.

Reply:
Done